### PR TITLE
Create win-run.json

### PR DESCRIPTION
**Pull request by iammahesh-dev**
## Cheat sheet for windows run

ia: https://duck.co/ia/view/windows_run_cheat_sheet

### CODE REVIEW

**Comment by iammahesh-dev** [comment]
@jophab Plz check this one and let me know if there is any error.

File: share/goodie/cheat_sheets/json/win-run.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{
+  "id": "windows_run_cheat_sheet",
+  "name": "Windows Run",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/win-run.json:3]: @mahi30795 "Windows Run Command" would be better.

File: share/goodie/cheat_sheets/json/win-run.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
+  "id": "windows_run_cheat_sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/win-run.json:2]: @mahi30795 windows_run_command_cheat_sheet would be better.
Also for file name make the same correction

File: share/goodie/cheat_sheets/json/win-run.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  "name": "Windows Run",
+  "description": "A single-line command line interface used to directly open an application or document whose path is known.",
+  "metadata": {
+    "sourceName": "Symantec",
+    "sourceUrl": "http://www.symantec.com/connect/articles/list-run-commands-windows-7-and-8"
+  },
+  "aliases": [
+   "run"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/win-run.json:10]: @mahi30795 lets make this "windows run"

**Comment by iammahesh-dev** [comment]
@jophab thanks for pointers.

**Pull request merged**